sklearn/model_selection/_split.py

```diff
@@ -1113,12 +1112,16 @@ class TimeSeriesSplit(_BaseKFold):
 
     Parameters
     ----------
-    n_splits : int, default=5
-        Number of splits. Must be at least 2.
+    n_splits : "walk_forward" or int, default=5
+        Number of splits. Must be at least 2. If `"walk_forward"`, the number
+        of splits is automatically set to obtain a rolling window.
```

Comment:
what does it mean exactly?

As in, if I have a series of length 1000, and I want 5 folds, but in a walk forward kinda way, what should I do? I don't get that information from the docstring.

It seems a bit impractical as the arguments are right now in this PR.

In practice, I'd like to be able to keep doing my 5 fold cross validation (take it computation constraints), but I'd like them to be in a walk forward way. The API right now doesn't let me easily do that.